parm/archive/gfsa.yaml.j2

```diff
@@ -0,0 +1,62 @@
+gfsa:
+    {% set head = "gfs.t" + cycle_HH + "z." %}
+    name: "GFSA"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gfsa.tar"
+    required:
+        # TODO explicitly name all logs to include
+        {% for log in glob("logs/" ~ cycle_YMDH ~ "/gfs*.log") %}
+        {% if log != "gfsarch.log" %}
+        - "{{ log }}"
+        {% endif %}
+        {% endfor %}
+        #    - "logs/{{ cycle_YMDH }}/{{ RUN }}fcst.log"
+        #    - "logs/{{ cycle_YMDH }}/{{ RUN }}atmos_prod_f*.log"
+        - "{{ COM_CONF | relpath(ROTDIR) }}/ufs.input.nml"
+        {% if MODE == "cycled" %}
+        - "{{ COM_ATMOS_GRIB_0p25 | relpath(ROTDIR) }}/{{ head }}pgrb2.0p25.anl"
+        - "{{ COM_ATMOS_GRIB_0p25 | relpath(ROTDIR) }}/{{ head }}pgrb2.0p25.anl.idx"
+        - "{{ COM_ATMOS_GRIB_1p00 | relpath(ROTDIR) }}/{{ head }}pgrb2.1p00.anl"
+        - "{{ COM_ATMOS_GRIB_1p00 | relpath(ROTDIR) }}/{{ head }}pgrb2.1p00.anl.idx"
+        #    - "logs/{{ cycle_YMDH }}/{{ RUN }}atmanlprod.log"
+        #    - "logs/{{ cycle_YMDH }}/{{ RUN }}prep.log"
+        #    - "logs/{{ cycle_YMDH }}/{{ RUN }}anal.log"
+        #    - "logs/{{ cycle_YMDH }}/{{ RUN }}atmanlupp.log"
+        {% if DO_VMINMON %}
+        - "{{ COM_ATMOS_MINMON | relpath(ROTDIR) }}/{{ cycle_YMDH }}.costs.txt"
+        - "{{ COM_ATMOS_MINMON | relpath(ROTDIR) }}/{{ cycle_YMDH }}.cost_terms.txt"
+        - "{{ COM_ATMOS_MINMON | relpath(ROTDIR) }}/{{ cycle_YMDH }}.gnorms.ieee_d"
+        - "{{ COM_ATMOS_MINMON | relpath(ROTDIR) }}/{{ cycle_YMDH }}.reduction.ieee_d"
+        - "{{ COM_ATMOS_MINMON | relpath(ROTDIR) }}/gnorm_data.txt"
+        #    - "logs/{{ cycle_YMDH }}/{{ RUN }}vminmon.log"
+        {% endif %}
+        {% endif %}  # Full cycle
+        {% for fhr in range(FHMIN_GFS, FHMAX_GFS + FHOUT_GFS, FHOUT_GFS) %}
+        - "{{ COM_ATMOS_GRIB_0p25 | relpath(ROTDIR) }}/{{ head }}pgrb2.0p25.f{{ '%03d' % fhr }}"
+        - "{{ COM_ATMOS_GRIB_0p25 | relpath(ROTDIR) }}/{{ head }}pgrb2.0p25.f{{ '%03d' % fhr }}.idx"
+        - "{{ COM_ATMOS_HISTORY | relpath(ROTDIR) }}/{{ head }}atm.logf{{ '%03d' % fhr }}.txt"
+        {% endfor %}
+        {% if AERO_ANL_CDUMP == "gdas" or AERO_ANL_CDUMP == "both" %}
+        - "{{ COM_CHEM_ANALYSIS | relpath(ROTDIR) }}/{{ head }}aerostat"
+        {% endif %}
+    optional:
+        {% if MODE == "cycled" %}
+        - "{{ COM_OBS | relpath(ROTDIR) }}/{{ head }}nsstbufr"
+        - "{{ COM_OBS | relpath(ROTDIR) }}/{{ head }}prepbufr"
+        - "{{ COM_OBS | relpath(ROTDIR) }}/{{ head }}prepbufr.acft_profiles"
+        {% if DO_JEDIATMVAR %}
+        - "{{ COM_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}atmvar.yaml"
+        - "{{ COM_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}atmstat"
+        {% else %}
+        - "{{ COM_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}gsistat"
```

Comment:
Same comment about the gsistat being required.  

@RussTreadon-NOAA: Should the atmvar.yaml and atmstat files similarly be required for DO_JEDIATMVAR=YES?